docs/pipelines.md

```diff
@@ -326,6 +326,8 @@ tasks:
 You can use [`PipelineResources`](#specifying-resources) as inputs and outputs for `Tasks`
```

Comment:
We should just delete this section since PipelineResources have been removed

Reply:
Good call. Should I get rid of it everywhere `PipelineResources` syntax is used?

Reply:
Yes please